### DISCUSSION TITLE
Add a multi-platform template

### DISCUSSION THREAD

**Discussion by MauroCoppola** [MonoGame/MonoGame#8571] [General]
Hi,
I started tinkering with Monogame some days ago making demo projects both for desktop and mobile and I saw that there isn't a template specific for this need. I know that Monogame is a framework and not an engine with its own conventions but why not offer even a template to set up a basic project structure without doing it manually every time?

**Comment by KelsamGames** [comment]
Because it is trivial to just add another project to your solution for your other target platform(s) and link it to any common code.
